webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/apidata/JsonResponseObjects.scala

```diff
@@ -870,6 +887,66 @@ object JsonResponseObjects {
     }
   }
 
+  /**
+   * Representation of a group category with bare minimum group information
+   */
+  final case class JRGroupCategoryInfo(
+      id:                                     String,
+      name:                                   String,
+      description:                            String,
+      @jsonField("categories") subCategories: List[JRGroupCategoryInfo],
+      groups:                                 List[JRGroupCategoryInfo.JRGroupInfo],
+      @jsonField("targets") targetInfos:      List[JRRuleTargetInfo]
+  )
+
+  object JRGroupCategoryInfo {
+    final case class JRGroupInfo(
+        id:                              NodeGroupId,
+        @jsonField("displayName") name:  String,
```

Comment:
for this one and other: why not using the json name directly ? Is it an choice given other constraints ?

Reply:
(I imagine with chimney, but if so: is there a reason to chose that way ?

Reply:
It avoid a boilerplate for the `Transformer` definition in chimney : we can derive the field transformation if the name is the same as the source datatype.  If we have the `displayName` field, we would have to write `withFieldRenamed(_.displayName, _.name)` (it is even more verbose if the transformer could be `derive`-d directly)

Reply:
but then, you wrote `@jsonField("displayName")̀` :) OK, there's less chars in that latter option. And perhaps it's a bit clearer.